aws/rds/main.tf

```diff
@@ -13,58 +13,90 @@ resource "aws_db_subnet_group" "this" {
   subnet_ids  = var.vpc.subnets
 }
 
-moved {
-  from = aws_db_subnet_group.this
-  to   = aws_db_subnet_group.this[0]
+resource "aws_db_parameter_group" "master" {
```

Comment:
I wouldn't add the parameter group with hardcoded parameters here. This affects the module reusability as every DB would have these set. Is there a good reason to have them like it?

I'm asking cause otherwise it would be better to leave people passing the parameter group as a parameter, this way these details are not in the module, and people would pass them from outside.